samples/excel/worksheet/worksheet-range-cell.yaml

```diff
@@ -0,0 +1,234 @@
+id: ''
+gist: ''
+source: EXCEL
+author: Microsoft
+name: Worksheet range and cell
+description: Explore the APIs to get a range or a cell in a worksheet
+script:
+  content: >
+    $("#setup").click(setup);
+
+
+    $("#get-used-range").click(getUsedRange);
+
+    $("#get-range").click(getRange);
+
+    $("#get-cell").click(getCell);
+
+
+    async function getUsedRange() {
+        try {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets.getActiveWorksheet();
+                let range = sheet.getUsedRange();
+                range.load("address");
+
+                await context.sync();
+
+                console.log(`The address of the used range in the worksheet is "${range.address}"`);
+            });
+
+            console.log("Done!");
+        }
+        catch (error) {
+            OfficeHelpers.Utilities.log(error);
+        }
+    }
+
+
+    async function getRange() {
+        try {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets.getActiveWorksheet();
+                let range = sheet.getRange();
```

Comment:
I think the sample would be better if getRange had an address.
e.g. getRange("A1").